pkg/provider/pulumi/azure/azure.go

```diff
@@ -195,12 +195,8 @@ func (a *azureProvider) Configure(ctx context.Context, autoStack *auto.Stack) er
 	return nil
 }
 
-func (a *azureProvider) ReadPulumiStack(ctx context.Context, backendURL string) (*common.PulumiStack, error) {
-	return nil, utils.NewNotSupportedErr("azure ReadPulumiStack()")
-}
-
-func (a *azureProvider) TryPullImage(ctx context.Context, imageURL string) error {
-	return utils.NewNotSupportedErr("azure TryPullImage()")
+func (a *azureProvider) TryPullImages() error {
```

Comment:
can you just totally remove the TryPullImages?

Reply:
@asalkeld there is still an implementation for TryPullImages for stack update that was implemented prior to these changes to try and fix ECR deletion.